### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by Aman-Jain1996**
Added wishlist page with jsx

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]
modular code
looks good to merge

**Review by shrey27 — APPROVED** [review]
overall code structure is very modular and readable
PR is good to merge

File: src/components/NavBar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+.Nav-header {
+  display: flex;
+  align-items: center;
+  background-color: var(--bg-lightWhite);
+  height: 12vh;
+  padding: 2.4rem 4.8rem;
+  justify-content: space-between;
+  position: sticky;
+  top: 0;
+  /* box-shadow: 0 4px 12px 0 rgba(0 , 0 ,0 , .8); */
```
Comment (Rohanmond) [inline src/components/NavBar/Navbar.css:12]: remove comments

File: src/pages/Product/Products.jsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  return (
+    <div>
+      <Navbar />
+      <div className="products-outer-container">
+        <Sidenav />
+        <section className="product-menu">
+          <span className="product-counter">
+            Showing All products (20 products)
+          </span>
+          <Link to="/products/id">
```
Comment (Rohanmond) [inline src/pages/Product/Products.jsx:19]: you could use onclick navigate here

**Review by Rohanmond — APPROVED** [review]

**Pull request merged**